tests/fv3_conf/cpld_control_run.IN

```diff
@@ -90,8 +90,13 @@ if [ $WARM_START = .false. ]; then
       cp    ${MOM_IC}/MOM.res.nc ./INPUT
       cp    ${ICE_IC}/cice_model_@[ICERES].cpc.res_* ./cice_model.res.nc
     else
-      cp    ${MOM_IC}/MOM6_IC_TS_${SYEAR}${SMONTH}${SDAY}${SHOUR}.nc ./INPUT/MOM6_IC_TS.nc
-      cp    ${ICE_IC}/cice_model_@[ICERES].cpc.res_${SYEAR}${SMONTH}${SDAY}.nc ./cice_model.res.nc
+      if [[ $OCNICE_WARMSTART == .true. ]]; then
+        cp   ${MOM_IC}/mom6.mx@[OCNRES].${SYEAR}${SMONTH}${SDAY}${SHOUR}.warmstart.nc ./INPUT/mom6.warmstart.nc
+        cp   ${ICE_IC}/cice.mx@[OCNRES].${SYEAR}${SMONTH}${SDAY}${SHOUR}.warmstart.nc ./cice.warmstart.nc
+      else
+        cp    ${MOM_IC}/MOM6_IC_TS_${SYEAR}${SMONTH}${SDAY}${SHOUR}.nc ./INPUT/MOM6_IC_TS.nc
+        cp    ${ICE_IC}/${OCNRES}/cice_model_@[ICERES].cpc.res_${SYEAR}${SMONTH}${SDAY}.nc ./cice_model.res.nc
```

Comment:
For lines 97-98, is OCNRES required for ICE ICs? Is it possible that OCNRES can be removed so that MOM_IC and ICE_IC directories will be consistent?

Reply:
I'd have to look again. Maybe.

Reply:
The issue here is that MOM IC and/or restart files that we have staged in either the input-data or the BMIC directories do not contain date or resolution information in the name. 

The 1deg and 1/2deg currently use a 1-deg global, rectilinear IC file, ``MOM6_IC_TS_2021032206.nc``. 

The 1/4deg restarts used by DATM and bmark are just 4 files name ``MOM.res.[].nc``. They contain no date or resolution information. There is also the``mom6_increment.nc`` file which has no date or resolution information but is used by the global applications (on the 1deg tripole) and one used by DATM test, also 1deg but a different date and stored in a different directory.

For CICE, all our tests use some version of a CICE restart and the file names contain both the resolution and the date---with the exception of the C48-5deg warm start test. 

There's also a disconnect between the ocean tripole resolution being ``mx[xxx]`` and the ice tripole resolution being ``x.xx``. 

Personally, I would prefer all files to have filenames which reflect the date and resolution rather than having to traverse a sub-directory structure. Making that change is not difficult, but it would be tedious.

Reply:
Thanks for the explanation. Let's leave it as is for this PR.